lighthouse-core/report/html/templates.html

```diff
@@ -681,19 +663,6 @@
 <!-- Lighthouse PWA badge gauge -->
 <template id="tmpl-lh-gauge--pwa">
   <style>
-    .lh-gauge--pwa__wrapper {
```

Comment:
you managed to have it share the styles?

Reply:
yup